core/market/decentralized/src/matcher/store.rs

```diff
@@ -66,7 +66,7 @@ impl SubscriptionStore {
                 inserted,
                 id,
             }),
-            Err(e) => Err(SaveOfferError::SaveError(e, id)),
+            Err(e) => Err(SaveOfferError::Save(e, id)),
```

Comment:
Maybe name it `DbError`? It looks strange and tells very little to reader

Reply:
I was considering `Internal` but the error message convinced me to leave it as is...